### PR TITLE
Prescription submission notebook

### PR DESCRIPTION
**Pull request by ofrancon**
Added a notebook to go through the prescriptor submission process

### CODE REVIEW

File: covid_xprize/validation/data/uniform_random_costs.csv [hunk 1]
```diff
@@ -23,33 +23,6 @@ Bolivia,,0.75,1.62,1.57,1.65,1.26,1.73,0.85,0.13,0.07,0.28,0.26,1.83
 Bosnia and Herzegovina,,0.72,1.15,1.48,1.51,1.27,1.0,1.27,0.71,1.28,0.55,0.78,0.27
 Botswana,,0.71,0.81,0.04,1.16,1.43,1.53,1.08,1.1,1.14,1.05,0.89,1.05
 Brazil,,0.95,1.12,0.86,1.08,0.94,0.92,0.81,1.61,1.33,0.61,0.32,1.45
-Brazil,Acre,0.21,1.03,0.79,0.87,1.55,1.24,1.71,0.5,1.05,0.24,1.77,1.05
```
Comment (ofrancon) [inline covid_xprize/validation/data/uniform_random_costs.csv:26]: Removed countries and regions that are not in `countries_regions.csv`

File: prescriptor_robojudge.ipynb [hunk 2]
```diff
@@ -169,7 +169,7 @@
     "#     'NeatExample': 'covid_xprize/examples/prescriptors/neat/test_prescriptions/pres.csv',\n",
     "    'Random1': 'covid_xprize/examples/prescriptors/random/prescriptions/random_presc_1.csv',\n",
     "    'Random2': 'covid_xprize/examples/prescriptors/random/prescriptions/random_presc_2.csv',\n",
-    "    'BlindGreedy': 'covid_xprize/examples/prescriptors/blind_greedy/prescriptions/blind_presc.csv',\n",
+    "    'BlindGreedy': 'covid_xprize/examples/prescriptors/blind_greedy/prescriptions/blind_greedy.csv',\n",
```
Comment (ofrancon) [inline prescriptor_robojudge.ipynb:172]: Wasn't matching the file created 1 cell above

File: prescriptor_submission_template.ipynb [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+    "            'H6_Facial Coverings']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def weight_prescriptions_by_cost(pres_df, cost_df):\n",
```
Comment (ofrancon) [inline prescriptor_submission_template.ipynb:304]: Duplicated some code from `prescriptor_robojudge.ipynb` for the sake of example. Probably need to factorize that code in something like `covid_xprize/score/prescriptor_scoring.py`. Later.

**Review by EKMeyerson — APPROVED** [review]
looks great; thanks

**Pull request merged**